test/com/google/enterprise/adaptor/AccumulatingDocIdPusher.java

```diff
@@ -60,8 +60,8 @@ public DocId pushNamedResources(Map<DocId, Acl> resources,
   @Override
   public GroupPrincipal pushGroupDefinitions(
       Map<GroupPrincipal, ? extends Collection<Principal>> defs,
-      boolean caseSensitive, ExceptionHandler handler)
-      throws InterruptedException {
+      boolean caseSensitive, boolean incremental, String sourceName,
+      ExceptionHandler exceptionHandler) throws InterruptedException {
```

Comment:
This could be a good time (in a separate CL) to look at sharing this class with the connectors, and possibly including a shared UnsupportedDocIdPusher (there's also DelegatingDocIdPusher, but it seems to be used only in implemented UnsupportedDocIdPusher).